ompi/mca/pml/ob1/pml_ob1_hdr.h

```diff
@@ -185,12 +178,6 @@ static inline void mca_pml_ob1_rget_hdr_prepare (mca_pml_ob1_rget_hdr_t *hdr, ui
 {
     mca_pml_ob1_rendezvous_hdr_prepare (&hdr->hdr_rndv, MCA_PML_OB1_HDR_TYPE_RGET, hdr_flags,
                                         hdr_ctx, hdr_src, hdr_tag, hdr_seq, hdr_msg_length, hdr_src_req);
-#if OPAL_ENABLE_HETEROGENEOUS_SUPPORT && OPAL_ENABLE_DEBUG
-    hdr->hdr_padding[0] = 0;
```

Comment:
Keep the zeroing of the padded structure when debug is enabled.

Reply:
He removed the padding from this struct. Haven't counted yet, but it might no longer be needed?

Reply:
You are right, the padding has been removed few lines above.